misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/evaluation/engine/FunctionNodeFinder.java

```diff
@@ -33,26 +33,32 @@
  *
  * @since 2.0.0
  */
-class FunctionNodeFinder extends NodeVisitor {
+public class FunctionNodeFinder extends NodeVisitor {
 
     private final String functionName;
     private FunctionDefinitionNode result;
 
-    FunctionNodeFinder(String functionName) {
+    public FunctionNodeFinder(String functionName) {
         this.functionName = functionName;
     }
 
-    public Optional<FunctionDefinitionNode> searchIn(Module module) {
+    /**
+     * Search for function definitions with the specified name, within the given ballerina module instance.
+     *
+     * @param module Ballerina module
+     * @return any function definitions with the specified name, if present.
+     */
+    public Optional<FunctionDefinitionNode> searchInModule(Module module) {
         for (DocumentId documentId : module.documentIds()) {
             if (result != null) {
                 break;
             }
-            searchIn(module.document(documentId));
+            searchInFile(module.document(documentId));
         }
         return Optional.ofNullable(result);
     }
 
-    public Optional<FunctionDefinitionNode> searchIn(Document document) {
+    public Optional<FunctionDefinitionNode> searchInFile(Document document) {
```

Comment:
Are we using the return value of `searchInFile` method ? If not, shall we make this method `void`?

Reply:
Refactored to use the return value in 8a8a248